<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Banquier</class>
 <widget class="QWidget" name="Banquier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>172</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9" stretch="1,3,0">
    <property name="spacing">
     <number>3</number>
    </property>
    <item>
     <widget class="QLabel" name="nom_agence">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Agence Mulhouse</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="photo">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/assets/pdp.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="username">
      <property name="font">
       <font>
        <family>MS PGothic</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>NOM</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>550</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QPushButton" name="ca_bours">
      <property name="text">
       <string>CA</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/ca.png</normaloff>:/assets/ca.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="vol_bours">
      <property name="text">
       <string>Volume</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/vol.png</normaloff>:/assets/vol.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pourc_bours">
      <property name="text">
       <string>Pourcentage</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/pourc.png</normaloff>:/assets/pourc.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QPushButton" name="vol_banq_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Volume</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/vol.png</normaloff>:/assets/vol.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ca_banq_2">
      <property name="text">
       <string>CA</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/ca.png</normaloff>:/assets/ca.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pourc_banq_2">
      <property name="text">
       <string>Pourcentage</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/pourc.png</normaloff>:/assets/pourc.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>550</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <property name="spacing">
     <number>7</number>
    </property>
    <item>
     <widget class="QPushButton" name="vol_assur_2">
      <property name="text">
       <string>Volume</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/vol.png</normaloff>:/assets/vol.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ca_assur">
      <property name="text">
       <string>CA</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/ca.png</normaloff>:/assets/ca.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pourc_assur">
      <property name="text">
       <string>Pourcentage</string>
      </property>
      <property name="icon">
       <iconset resource="ressource.qrc">
        <normaloff>:/assets/pourc.png</normaloff>:/assets/pourc.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolButton" name="elargir_bq_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>27</width>
     <height>26</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>elargir</string>
   </property>
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/assets/elargir_white.png</normaloff>:/assets/elargir_white.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="elargir_bours_2">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>260</y>
     <width>27</width>
     <height>26</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>elargir</string>
   </property>
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/assets/elargir_white.png</normaloff>:/assets/elargir_white.png</iconset>
   </property>
  </widget>
  <widget class="QtCharts::QChartView" name="produit_bq">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>311</width>
     <height>301</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QtCharts::QChartView" name="produit_bours">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>240</y>
     <width>311</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="elargir_assur_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>260</y>
     <width>27</width>
     <height>26</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>elargir</string>
   </property>
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/assets/elargir_white.png</normaloff>:/assets/elargir_white.png</iconset>
   </property>
  </widget>
  <widget class="QtCharts::QChartView" name="produit_assur">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>153</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="date_picker"/>
    </item>
   </layout>
  </widget>
  <zorder>produit_bours</zorder>
  <zorder>produit_assur</zorder>
  <zorder>produit_bq</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>layoutWidget_7</zorder>
  <zorder>layoutWidget_8</zorder>
  <zorder>elargir_bq_2</zorder>
  <zorder>elargir_bours_2</zorder>
  <zorder>elargir_assur_2</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qtcharts/qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
